quipucords/tests/api/credential/test_credential.py

```diff
@@ -1,4 +1,4 @@
-"""Test the API application."""
+"""Tests for quipucords.api.source.view."""
```

Comment:
I like that u are using `quipucords` as a main module name. I wish we weren't using default django organization and had it that way. some day!

Reply:
s/module/package/

Reply:
We'll consider this an aspirational comment for now. 😉 We would love to have saner package namespaces in the future. Maybe one day this string just automatically becomes correct.